ios/DocumentScannerView.m

```diff
@@ -63,14 +63,33 @@ - (void) capture {
                                     @"croppedImage": [croppedImageData base64EncodedStringWithOptions:NSDataBase64Encoding64CharacterLineLength],
                                     @"initialImage": [initialImageData base64EncodedStringWithOptions:NSDataBase64Encoding64CharacterLineLength],
                                     @"rectangleCoordinates": rectangleCoordinates });
+            } else if(self.saveOnDisk){
+               NSError *error;
+               NSFileManager *fileMgr = [NSFileManager defaultManager];
+               NSString *documents = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) objectAtIndex:0];
+               NSString *dataPath = [documents stringByAppendingPathComponent:@"PJBANK_DOCUMENTS"];
```

Comment:
This should probably either let the user specify a folder name or use the CFBundleDisplayName without spaces maybe?